cinema/urls.py

```diff
@@ -1,10 +1,64 @@
-from django.urls import path
+from django.urls import path, include
+from cinema.views import (
+    MovieViewSet,
+    CinemaHallViewSet
+)
+from rest_framework import routers
+from cinema.views import (
+    GenreList,
+    GenreDetail,
+    ActorList,
+    ActorDetail
+)
 
```

Comment:
Correct imports according to pep8

Reply:
if it is not changed, it does not work for me